README.md

```diff
@@ -5,22 +5,288 @@
+        ```
+
+    * Declare a part file `catalog.g.dart` in the begining of the `catalog.dart` dart file after all imports.
+
+        ```dart
+        import 'dart:io';
+
+        part 'catalog.g.dart'
+        ```
+
+    * Create a data model class
+
+        ```dart
+        @RealmModel()
+        class _Item {
+            @PrimaryKey()
+            late final int id;
+
+            late String name;
+            
+            int price = 42;
+        }
+        ```
+    * Generate RealmObject class `Item` from data model class `_Item`.
+
+        ```
+        flutter run realm generate
+        ```
+        A new file `catalog.g.dart` will be created next to the `catalog.dart`.
+        
+        _*This file should be committed to source control_
+
+    * Use the RealmObject class `Item` with Realm.
+
+        ```dart
+        // Create a Configuration object
+        var config = Configuration(Item.schema);
+
+        // Opean a Realm
+        realm = Realm(config);
+
+        // Open a write transaction
+        realm.write(() {
+          var myItem = Item(0, 'Iteam', price: 4);
+          realm.add(myItem);
+          final item = realm.add(Item(1, 'Iteam')..price = 20);
+        });
+
+        // Get objects from the realm
+
+        // Get all objects
+        var items = realm.all<Item>();
+        
+        // Get object by index
+        var item = items[5];
+        
+        // Get object by primary key
+        var itemByKey = realm.find<Item>(0);
+        
+        // Filter and sort object
+        var objects = realm.query<Item>("name == 'Special Item'");
+        var mymake = 'Tesla';
+        var objects = realm.query<Item>(r'make == $0', [mymake]");
+        ```
+
+# Realm Dart SDK 
+
+The Realm Dart package is `realm_dart`
+
+## Environment setup for Realm Dart
+
+* Supported platforms are Windows and Mac.
+
+* Dart SDK ^2.15
+
+## Usage
+
+* Add `realm_dart` package to a Dart application.
+
+    ```
+    dart pub add realm_dart
+    ```
+
+* Install the `realm_dart` package into the application. This downloads and copies the required native binaries to the app directory
+
+    ```
+    dart run realm_dart install
+    ``` 
+
+* Enable generation of Realm schema objects.
+
+    * Add `build_runner` package to `dev_dependencies`.
+
+        ```
+        dart pub add build_runner --dev
+        ```
+    * Enable realm_generator by adding a `build.yaml` file to the application.
+
+        ```yaml
+        targets:
+            $default:
+                builders:
+                    realm_generator|realm_object_builder:
+                        enabled: true
+                        generate_for:
+                            - bin/*.dart 
+        ```
+
+## Usage
+
+For usage of Realm Dart see the Realm Flutter usage above
+
+
+# Building the source
+
+## Building Realm Flutter
+
+    * clone the repo 
+    ```
+    git clone https://github.com/realm/realm-dart
+    git submodule udpate --init --recursive
+    ```
```

Comment:
```suggestion
* clone the repo
    ```
    git clone https://github.com/realm/realm-dart
    git submodule udpate --init --recursive
    ```
```

Reply:
Moved tabulation because after generating the doc asterisk and "`" were displayed instead of ballet and code formatting.

Reply:
fixed. good catch